api/v1alpha1/types_etcd.go

```diff
@@ -178,6 +178,13 @@ type BackupSpec struct {
 	// DeltaSnapshotMemoryLimit defines the memory limit after which delta snapshots will be taken
 	// +optional
 	DeltaSnapshotMemoryLimit *resource.Quantity `json:"deltaSnapshotMemoryLimit,omitempty"`
+	// DeltaSnapshotRetentionPeriod defines the duration for which delta snapshots will be retained, excluding the latest snapshot set.
+	// The value should be a string formatted as a duration (e.g., '1s', '2m', '3h', '4d') with a minimum value of 1 second.
```

Comment:
Why do we set minimum value as `1s`? If this field is not set at all (since it's `optional`), then the default value for this would be 0 in etcd-backup-restore, according to https://github.com/gardener/etcd-backup-restore/blob/master/doc/usage/garbage_collection.md. So why would we want to disallow setting `0s` in druid then?